keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -111,6 +118,43 @@ For the first implementation we do not want to support restore in the
 outside of Kubernetes. The restore is a container engine only operation
 in this first step.
 
+A high level view on the implementation is that triggering the *kubelet* API
+endpoint will trigger the `ContainerCheckpoint` CRI API endpoint to create a
+checkpoint at the location defined by the *kubelet*.  In the checkpoint request
+the kubelet will specify the name of the checkpoint archive as
+`checkpoint-<podFullName>-<containerName>-<timestamp>.tar` and also request to
+store the checkpoint archive in the `checkpoints` directory below its root
+directory (as defined by `--root-dir`). This defaults to
+`/var/lib/kubelet/checkpoints`.
+
+To trigger a checkpoint following HTTP Request has to be made against the *kubelet*:
+
+- `POST /checkpoint/{namespace}/{pod}/{container}``
```

Comment:
What RBAC check does the kubelet make on this endpoint now? there will be a resource, name, verb tuple that it tries for each endpoint.  I don't recall anywhere in the kubelet doing * on *

Reply:
@kannon92 @mrunalp can you provide some details how the kubelet handles this?